webrtc.html

```diff
@@ -866,7 +866,7 @@
 
                   <li>
                     <p>If <var>connection</var>'s <a>signaling state</a>
-                    changed above, fire a simple event named <code><a href=
+                    changed above, queue a task to fire a simple event named <code><a href=
```

Comment:
We have had issues with the state being overwritten by a second event after the task was queued but before the event was fired, so yes, I think we should set the state within the queued task if we queue it (otherwise, the user observes a surprising state transition).